### DISCUSSION TITLE
Error while installating FaceRecognition app.

### DISCUSSION THREAD

**Discussion by apoorv569** [nextcloud/all-in-one#1551] [Questions] [answered]
While trying to install the FaceRecognition app I am getting this error,
```
App "Face Recognition" cannot be installed because the following dependencies are not fulfilled: The library pdlib is not available
```

How can I install the missing dependency?

**Comment by szaimen** [comment]
Hi, we have this feature available for adding php extensions: https://github.com/nextcloud/all-in-one#how-to-add-php-extensions-permanently-to-the-nextcloud-container

**Reply by apoorv569** [reply]
> Hi, we have this feature available for adding php extensions: https://github.com/nextcloud/all-in-one#how-to-add-php-extensions-permanently-to-the-nextcloud-container

I see, the website mentioned in the link you shared, I can't find `pdlib`..

https://pecl.php.net/package-search.php?pkg_name=pdlib&pkg_maintainer=&pkg_category=&released_on_year=&released_on_month=&released_on_day=&released_before_year=&released_before_month=&released_before_day=&released_since_year=&released_since_month=&released_since_day=

**Comment by szaimen** [comment]
Ah I see, so this extension does not seem to be available on PECL nor via PHP core. 

You can request this here: https://github.com/goodspb/pdlib/issues/new.

**Reply by apoorv569** [reply]
> Ah I see, so this extension does not seem to be available on PECL nor via PHP core.
> 
> You can request this here: https://github.com/goodspb/pdlib/issues/new.

OK, I  opened new issue for requesting this.

https://github.com/goodspb/pdlib/issues/56

**Comment by szaimen** [comment, marked as answer]
I just added this note to the readme: https://github.com/nextcloud/all-in-one#what-about-the-pdlib-php-extension-for-the-facerecognition-app
